README.md

```diff
@@ -1 +1,59 @@
-# angular-project-setup-lab
+# Project Setup
+
+## Objectives
+
+- Download the latest version of Angular 1.x
+- Setup the directory structure for your Angular application
+ 
+## Instructions
+
+Go to the [Angular website](https://angularjs.org/) and download the latest version.
+
+Setup your directory structure. From the root directory, the structure should be as follows (you can find the index.html in this repo) -
+
+- js/
+  - app/
+    - directives/
+    - controllers/
+    - services/
+    - templates/
+      - partials/
+      - views/
+  - angular.js
+- img/
+- css/
+- index.html
+
+Make sure you put the Angular code you downloaded into the `js/` folder!
+
+Let's create some files so we can have something pretty displayed in browser. No need to worry with what this code does yet, we will be covering it all very shortly!
```

Comment:
Really nice context here - like that you're getting the students coding a little bit